docs/apis/http/tus_upload.md

```diff
@@ -0,0 +1,178 @@
+---
+title: "tus upload"
+date: 2023-10-10T00:00:00+00:00
+weight: 21
+geekdocRepo: https://github.com/owncloud/ocis
+geekdocEditPath: edit/master/docs/apis/http
+geekdocFilePath: tus_upload.md
+geekdocCollapseSection: true
+---
+
+Infinite Scale supports [tus resumable-upload protocol](https://tus.io/protocols/resumable-upload).
+tus is a robust, modular, and open protocol designed to resume large file uploads reliably over HTTP.
+In situations where file uploads might be interrupted due to network issues, browser crashes, or other unforeseen interruptions,
+tus ensures that uploads can be resumed from the point of failure without losing data.
+
```

Comment:
please mention that this documentation shows some basic examples and that more details about the tus protocol can be found in their own docs